summarization/stamp_page_picking/scoring_utils.py

```diff
@@ -0,0 +1,146 @@
+''' Module for scoring utils '''
+
+from sklearn.metrics.pairwise import cosine_similarity
+
```

Comment:
Can we have a separate class to calculate each score?

Reply:
Adding this as a TODO